freqtrade/arguments.py

```diff
@@ -0,0 +1,251 @@
+"""
+This module contains the argument manager class
+"""
+
+import argparse
+import os
+import re
+import logging
+from typing import List
+
+from freqtrade import __version__
+from freqtrade.constants import Constants
+
+
+class Arguments(object):
+    """
+    Arguments Class. Manage the arguments received by the cli
+    """
+
+    def __init__(self, args: List[str], description: str):
+        self.args = args
+        self.parsed_arg = None
+        self.parser = argparse.ArgumentParser(description=description)
+
+    def _load_args(self):
+        self.common_args_parser()
+        self._build_subcommands()
+
+    def get_parsed_arg(self) -> List[str]:
```

Comment:
Just a minor thing, but this method could be refactored into a property, which is more pythonic.
i.e.:
```
    @property
    def parsed_arg(self) -> argparse.Namespace:
        """
        Return the list of arguments
        :return: List[str] List of arguments
        """
        if self._parsed_arg is None:
            self.common_args_parser()
            self._build_subcommands()
            self._parsed_arg = self.parse_args()

        return self._parsed_arg
```

I also would drop `_load_args()` and do the initialization inline, because it is only used in one place. So we can reduce some LOCs